src/json/webhooks/UserCreatedClassicDisabled.schema.json

```diff
@@ -94,6 +95,7 @@
       }
     },
     "programShareLinks": {
+      "description": "The users primary share links for the primary code of each program",
```

Comment:
@DerekSiemens I don't think is valid schema? I think if you use `$ref` like this you have to use `allOf` too?